src/GeekPlatform/ViewModels/Jelentkezes/JelekezesViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using GeekPlatform.Models;
+
+namespace GeekPlatform.ViewModels.Jelentkezes
+{
+    public class JelekezesViewModel : ViewModelBase
+    {
+        public JelekezesViewModel(IEnumerable<Course> courselist)
+        {
+            Courses = courselist.Select(x => new CourseViewModel(x)).ToList();
```

Comment:
Szerintem itt tegyük hozzá `CourseEnrollment`-ből, hogy már jelentkezett-e a kurzusra, és írjuk ki View-ba is, **vagy** csak Controllerben szűrjük ki a már bejelentkezett kurzusokat, és hide